platform-sdk/swirlds-benchmarks/src/jmh/java/com/swirlds/benchmark/reconnect/MerkleBenchmarkUtils.java

```diff
@@ -118,8 +118,8 @@ private static <T extends MerkleNode> T testSynchronization(
                 firstReconnectException.compareAndSet(null, t);
                 return false;
             };
-            final StandardWorkGroup workGroup =
-                    new StandardWorkGroup(getStaticThreadManager(), "synchronization-test", null, exceptionListener);
+            final StandardWorkGroup workGroup = new StandardWorkGroup(
+                    getStaticThreadManager(), "synchronization-test", null, exceptionListener, false);
```

Comment:
This change (and a few more below) is redundant, as `logExceptionsToStdErr` is false by default.

Reply:
Yes, that was my initial hope. However,  it didn't work this way, unfortunately. The first constructor contains 3 arguments and the second contains 5. So, without this change the code doesn't compile. Alternatively, I can create another constructor but i thought it would be easier to just provide values.

Reply:
Nvm. Added another constructor